### PR TITLE
new double down build and mesh tally arg names

### PR DESCRIPTION
**Pull request by Shimwell**
## Proposed changes

Keen to get this PR merged in as the I am keen to renamed mesh_tally args before anyone starts using them instead of renaming them in a few weeks and cause more trouble for any users. They are a relatively new feature so I think this tiny breaking change will be unnoticed.

- Changes to the NeutronicsModel arguments to make them conform to pep8. 
    - mesh_tally_2D is now mesh_tally_2d
    - mesh_tally_3D is now mesh_tally_3d

- Minor changes to the dockerfile
    - .git has been added to the endings of cloned repos where it was missing. Thanks to @AI-Pranto for spotting that on the workshop repo
    - double-down is now using the /embree folder as discussed https://github.com/openmc-dev/openmc/issues/1749

## Types of changes

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code refactoring
- [x] Documentation Update (if none of the other choices apply)
- [ ] New tests

## Checklist

- [ ] Pep8 applied
- [x] Unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Comment by RemDelaporteMathurin** [comment]
@Shimwell any idea why the GA CI fails ?

File: paramak/parametric_neutronics/neutronics_model.py [hunk 1]
```diff
@@ -56,8 +56,10 @@ class NeutronicsModel():
             neutronics-material-maker.MultiMaterial. All components within the
             geometry object must be accounted for. Material tags required
             for a Reactor or Shape can be obtained with .material_tags.
-        mesh_tally_2D (list of strings): the mesh based tallies to calculate,
-            options include tritium_production, heating and flux
+        mesh_tally_2d (list of strings): the 2D mesh based tallies to calculate,
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_neutronics/neutronics_model.py:59]: wouldn't `(list of str)` be more appropriate ?

File: paramak/parametric_neutronics/neutronics_model.py [hunk 2]
```diff
@@ -56,8 +56,10 @@ class NeutronicsModel():
...
+            options include tritium_production, heating and flux.
+        mesh_tally_3d (list of strings): the 3D mesh based tallies to calculate,
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_neutronics/neutronics_model.py:61]: wouldn't `(list of str)` be more appropriate ?

File: paramak/parametric_neutronics/neutronics_model.py [hunk 3]
```diff
@@ -69,10 +71,14 @@ class NeutronicsModel():
             Defaults to 1e-4.
         faceting_tolerance (float): the tolerance to use when faceting surfaces.
             Defaults to 1e-1.
-        mesh_2D_resolution (tuple of ints): The mesh resolution in the height
+        mesh_2D_resolution (tuple of ints): The 3D mesh resolution in the height
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_neutronics/neutronics_model.py:74]: wouldn't `(tuple of int)` be more appropriate ?

File: paramak/parametric_neutronics/neutronics_model.py [hunk 4]
```diff
@@ -69,10 +71,14 @@ class NeutronicsModel():
...
             and width directions. The larger the resolution the finer the mesh
             and more computational intensity is required to converge each mesh
             element.
+        mesh_3D_resolution (tuple of ints): The 3D mesh resolution in the height,
```
Comment (RemDelaporteMathurin) [inline paramak/parametric_neutronics/neutronics_model.py:78]: wouldn't `(tuiple of int)` be more appropriate ?

**Review by RemDelaporteMathurin — COMMENTED** [review]
Perhaps renaming the PR to something more meaningful since there are only a few changes ? 😄 

A few remarks on the types in docstrings

File: paramak/parametric_neutronics/neutronics_model.py (hunk 1 above)
Comment (Shimwell) [inline paramak/parametric_neutronics/neutronics_model.py:59]:
```suggestion
        mesh_tally_2d (list of str): the 2D mesh based tallies to calculate,
```

File: paramak/parametric_neutronics/neutronics_model.py (hunk 2 above)
Comment (Shimwell) [inline paramak/parametric_neutronics/neutronics_model.py:61]:
```suggestion
        mesh_tally_3d (list of str): the 3D mesh based tallies to calculate,
```

File: paramak/parametric_neutronics/neutronics_model.py (hunk 4 above)
Comment (Shimwell) [inline paramak/parametric_neutronics/neutronics_model.py:78]:
```suggestion
        mesh_3D_resolution (tuple of int): The 3D mesh resolution in the height,
```

**Comment by Shimwell** [comment]
Thanks for the docstring improvments @RemDelaporteMathurin 

I guess these formatting errors would be fixed by using type hinting as well

**Comment by Shimwell** [comment]
Circle CI is failing now with the error ```No such file or directory: 'openmc'``` so it looks like the dependency image has not build properly

**Comment by Shimwell** [comment]
Full sweep of green ticks, I think this one is ready to merge

**Review by RemDelaporteMathurin — APPROVED** [review]

**Pull request merged**